### PR TITLE
JBPM-8247 - ColumnMeta should be ordered when adding new columns

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch @ederign @jstastny-cz please review this as it is a blocker, the Task Inbox throws an exception now when first listing the tasks.

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

File: uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java [hunk 1]
```diff
@@ -189,15 +183,32 @@ public void testAddColumnBeforeActionsOnAddColumn(){
...
         columnPicker.addColumn(comment);
         assertEquals(5, columnPicker.getDataGrid().getColumnCount());
         assertEquals("name", columnPicker.getDataGrid().getColumn(0).getDataStoreName());
         assertEquals("age", columnPicker.getDataGrid().getColumn(1).getDataStoreName());
         assertEquals("etc", columnPicker.getDataGrid().getColumn(2).getDataStoreName());
         assertEquals("comment", columnPicker.getDataGrid().getColumn(3).getDataStoreName());
         assertEquals("actions", columnPicker.getDataGrid().getColumn(4).getDataStoreName());
+    }
+
+    @Test
```
Comment (jstastny-cz) [inline uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java]: Formatting of the newly added method.

Comment (cristianonicolai) [reply uberfire-extensions/uberfire-widgets/uberfire-widgets-table/src/test/java/org/uberfire/ext/widgets/table/client/UberfireColumnPickerTest.java]: @jstastny-cz fixed

**Comment by jstastny-cz** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
@kiegroup/foundation-gatekeeper can any of you please merge this?

**Comment by tiagobento** [comment]
Merged as requested by @lazarotti.

**Pull request merged**